torchci/lib/bot/autoLabelBot.ts

```diff
@@ -165,6 +167,32 @@ export async function getLabelsFromLabelerConfig(
   return labels;
 }
 
+export async function getLabelsFromLabelToLabelConfig(
```

Comment:
Do you have an example of label2label config at hand so I could take a look?  I check malfet/deleteme repo, but I don't think it's there yet

Reply:
The tests should have an example